modules/docs/src/04-codegen/01-customisation.md

```diff
@@ -325,3 +325,91 @@ string Email
 By default, smithy4s renders collection types as unwrapped EXCEPT when the collection has been refined. In this case, the collection will be rendered within a newtype by default. If you wish your refined collection be rendered unwrapped, you can accomplish this using the same `@unwrap` trait annotation on it.
 
 :::
+
+## Default rendering
+
+Smithy4s allows you to customize how defaults on the fields of smithy structures are rendered inside of case classes. There are three options:
+
+- `FULL`
+- `OPTION_ONLY`
+- `NONE`
+
+The default is `FULL`.
+
+This value is set using metadata which means that the setting will be applied to all the rendering done by smithy4s.
+
+#### FULL
+
+`FULL` means that default values are rendered for all field types. For example:
+
+```kotlin
+metadata defaultRenderMode = "FULL"
```

Comment:
All the metadata bits that smithy4s surfaces/uses should be prefixed with `smithy4s` one way or another. If `.` are allowed in metadata keys, we could make it `smithy4s.defaultRenderMode`

Reply:
Unfortunately, dots aren't allowed. But I changed it to `smithy4sDefaultRenderMode`